core/src/remote/api/api.cpp

```diff
@@ -14,6 +14,8 @@
 #include <QJsonObject>
 #include <QJsonArray>
 
+using namespace remote::api;
```

Comment:
This may be not needed since on line 32 you're using a fully qualified name.
`using namespace` directives may be dangerous. If it's not necessary, please remove it.